core/src/main/scala/org/apache/spark/rdd/NewHadoopRDD.scala

```diff
@@ -230,12 +251,16 @@ class NewHadoopRDD[K, V](
     super.persist(storageLevel)
   }
 
-
-  def getConf: Configuration = confBroadcast.value.value
```

Comment:
This old `getConf` used to be public, but now it's protected. The old `getConf` used to always return the same object, whereas the new `getConf` will return different Configuration instances on back-to-back calls. I don't know enough about how developers use NewHadoopRDD directly to be able to say whether this would be a problem.